requirements.txt

```diff
@@ -3,7 +3,7 @@ beautifulsoup4==4.9.3
 bs4==0.0.1
 certifi==2020.12.5
 chardet==4.0.0
-django==5.0
+django==5.1
```

Comment:
Oppdaterer til 5.1 for å kunne bruke `query_params` i test-klienten:

https://docs.djangoproject.com/en/5.1/topics/testing/tools/#making-requests